src/accel/detail/IntegrationSingleton.hh

```diff
@@ -81,7 +90,8 @@ class IntegrationSingleton
     Mode mode_{Mode::size_};
     SetupOptions options_;
     SharedParams params_;
-    SetupOptionsMessenger messenger_{&options_};
+    std::unique_ptr<ScopedMpiInit> scoped_mpi_;
+    std::unique_ptr<SetupOptionsMessenger> messenger_;
```

Comment:
Why is the `SetupOptionsMessenger` becoming a `unique_ptr`?

Reply:
To delay initialization and allow us to wrap construction in a G4exception